src/components/Hero.astro

```diff
@@ -0,0 +1,29 @@
+---
+import houstonOmg from "../assets/houston_omg.webp"
+---
+
+<section class="flex flex-col items-center gap-16 py-16 lg:flex-row">
+	<h1 class="heading-2 text-center lg:text-left">It’s go time with Astro</h1>
+	<div class="rounded-md bg-blue-purple-gradient p-px">
+		<section
+			class="flex flex-col gap-4 rounded-md bg-astro-gray-700 px-6 shadow-md lg:flex-row"
+		>
+			<div class="flex flex-1 flex-col gap-4 py-6 text-astro-gray-200">
+				<h2 class="font-bold text-astro-gray-100">Let’s get started</h2>
+				<p>
+					Ready to get started with an Astro project but want to get to building
+					right away instead of spend time setting up your local environment?
```

Comment:
I'd love for @FredKSchott to look more closely at this text. 

I feel like the opening paragraph might be a bit of a mouthful. I think we can go shorter and snappier, if we want! And leave the details to the paragraph below.... which *could* cut "right in your browser" if we felt like it? Especially if the mention of "in your browser" gets used in this opening line.

Some possible lines/phrases (also consider mixing/matching):

- Want to try Astro without setting up a local environment?
- Ready to get started?
- Play around with a full, working version of Astro right in your browser!
- Prepare for lift-off: launch an Astro template, right in your browser!